include/channels.h

```diff
@@ -43,7 +43,7 @@ class CoreExport Channel final
 public:
 	/** A map of Memberships on a channel keyed by User pointers
 	 */
-	typedef std::map<User*, insp::aligned_storage<Membership>> MemberMap;
+	typedef std::unordered_map<User*, insp::aligned_storage<Membership>> MemberMap;
```

Comment:
I think changing this will cause any user list (e.g. `/NAMES`) to no longer be in alphabetical order.

Reply:
std::map in this case would use `std::less<User*>` which does direct pointer value compares and would be in arbitrary order.

I checked `/names #inspircd.dev` on irc.teranova.net (InspIRCd-4) and confirmed it wasn't alphabetical without this change.